ansible/roles/ocp-workload-bxms-dm-foundations/tasks/wait_for_deploy.yml

```diff
@@ -6,15 +6,21 @@
 #
 # Manual Test to determine list of unready replication controllers :
 #  1) install jp :  https://github.com/jmespath/jp
-#  2) oc get rc -o json | jp 'items[?  (status.readyReplicas == ""|| status.readyReplicas == `0`) ].metadata.annotations."openshift.io/deployment-config.name"'
+#  2) oc get rc -o json | jp 'items[?
+#    (status.readyReplicas == ""|| status.readyReplicas == `0`)
+#    ].metadata.annotations."openshift.io/deployment-config.name"'
 #
-- name: "Wait for following deployments to become ready: {{pod_to_wait}}"
+- name: "Wait for following deployments to become ready: {{ pod_to_wait }}"
   command: 'oc get rc -o json -n "{{ ocp_project }}"'
   register: rc_state
   changed_when: false
   retries: "{{ deploy_status_retries }}"
   delay: "{{ deploy_status_delay }}"
-  until: 'rc_state.stdout |from_json |json_query(''items[?  (status.readyReplicas == ""|| status.readyReplicas == `0`) ].metadata.annotations."openshift.io/deployment-config.name"'') |intersect(pod_to_wait) |length == 0'
+  until: >-
+    rc_state.stdout | from_json | json_query(''items[?
+    (status.readyReplicas == ""|| status.readyReplicas == `0`)
+    ].metadata.annotations."openshift.io/deployment-config.name"'')
+    | intersect(pod_to_wait) | length == 0
```

Comment:
This is still quite hard to read. The `json_query` content should be made into a task var to make this easier to understand.